ansible/playbooks/Supporting_Scripts/Nagios_Ansible_Config_tool/Nagios_Ansible_Config_tool.sh

```diff
@@ -110,9 +110,13 @@ Sys_Icon="yes"
 # Test: If client $Arch {{ ansible_architecture }} is aarch64 or armv7l then set the Nagios icon to "arm", else use $Distro {{ ansible_distribution }}
 if [[ $Arch = "aarch64" ]] || [[ $Arch = "armv7l" ]] ; then
         Sys_Icon_Picked=arm
+elif [[ $Distro = "MacOSX" ]] ; then
+	Sys_Icon_Picked="apple"
 else
         Sys_Icon_Picked=$Distro
 fi
+
+fi
```

Comment:
this `fi` doesn't look like it's in the right place?

Reply:
Resolved :+1: